.github/workflows/uclid-verifier-c-tests.yml

```diff
@@ -0,0 +1,69 @@
+name: Uclid5-based Verifier Tests
+
+on:
+  # Trigger this workflow also on workflow_call events.
+  workflow_call:
+
+env:
+  ACTIONS_STEP_DEBUG: TRUE
+  ACTIONS_RUNNER_DEBUG: TRUE
+
+jobs:
+  run:
+    strategy:
+      matrix:
+        platform: [ubuntu-latest]
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - name: Check out lingua-franca repository
+        uses: actions/checkout@v3
+        with:
+          path: lingua-franca
+          submodules: true
+          fetch-depth: 0
+      - name: Check out lf-verifier-benchmarks repository
+        uses: actions/checkout@v3
+        with:
+          repository: lsk567/lf-verifier-benchmarks
+          ref: main
+          path: lf-verifier-benchmarks
+      - name: Check out Uclid5 repository
+        uses: actions/checkout@v3
+        with:
+          repository: uclid-org/uclid
+          ref: 4fd5e566c5f87b052f92e9b23723a85e1c4d8c1c
+          path: uclid
+      - name: Set up JDK 17
```

Comment:
I vaguely recall we stopped using that action due to flakiness, but that might be averted by the caching we do (except on Windows, where it might rear its head because don't have caching working on Windows.

Reply:
OK, but if we start seeing flakiness in this step, the action item is to change back to using the method that we currently use in `prepare-build-env` to select the correct Java version.